rust/src/wrapper/workbook.rs

```diff
@@ -34,6 +34,7 @@ impl Workbook {
     /// TODO: example omitted
     #[wasm_bindgen(constructor)]
     pub fn new() -> Self {
+        console_error_panic_hook::set_once();
```

Comment:
How do you think of calling `std::panic_set_hook` in the default export (xlsxInit) instead?
```rust
std::panic::set_hook(Box::new(console_error_panic_hook::hook));
```

Reply:
Thanks! I'll do so.